client/my-sites/backup/backup-contents-page/file-browser/file-browser-node.tsx

```diff
@@ -23,7 +24,6 @@ const FileBrowserNode: FunctionComponent< FileBrowserNodeProps > = ( {
 	isAlternate,
 } ) => {
 	const isRoot = path === '/';
-	const hasChildren = item.hasChildren;
```

Comment:
This `hasChildren` constant was redundant. We could use `item.hasChildren` directly in the two instances we need it in this component.